Migrations/MyPlaceDbContext.cs

```diff
@@ -0,0 +1,27 @@
+using rexfinder_api.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace rexfinder_api.Migrations;
+
+public class MyPlaceDbContext : DbContext
+{
+    public DbSet<MyPlace> MyPlaces { get; set; }
+    public MyPlaceDbContext(DbContextOptions<MyPlaceDbContext> options)
+        : base(options)
+    {
+    }
+
+    protected override void OnModelCreating(ModelBuilder modelBuilder) {
+        base.OnModelCreating(modelBuilder);
+
+        modelBuilder.Entity<MyPlace>(entity =>
+        {
+            entity.HasKey(e => e.MyPlaceId);
+            entity.Property(e => e.Visited);
+            entity.Property(e => e.UserId);
+            entity.Property(e => e.CreatedOn).IsRequired();
```

Comment:
I don't know if this should be required?

Reply:
Probably not, I was just trying to make it consistent with the data model, where it is required.